projects/lendai-us/design-system/src/lib/steper/step-label/step-label.component.scss

```diff
@@ -0,0 +1,127 @@
+@use 'color' as c;
+
+$sizes: (
+  'md': 9px 16px,
+  'lg': 13px 20px,
+);
+
+:host {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  box-sizing: border-box;
+  counter-increment: counter 1;
+
+  &:not(:first-child)::before {
+    content: "";
+    background-image: url("data:image/svg+xml,%3Csvg width='5' height='62' viewBox='0 0 5 62' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cg clip-path='url(%23clip0_28_1592)'%3E%3Cline x1='2.51083' y1='-12.0893' x2='2.51081' y2='386.089' stroke='%23D9C5FF' stroke-width='1.82145' stroke-linecap='round' stroke-dasharray='9.11 9.11'/%3E%3C/g%3E%3Cdefs%3E%3CclipPath id='clip0_28_1592'%3E%3Crect width='5' height='62' fill='white'/%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E");
```

Comment:
can we export it as a regular image instead of data uri? just to be more consistent with other components